### ISSUE TITLE
All content should be contained within landmarks

### ISSUE THREAD

**Issue by aitchiss** [code-newbies/codeland#51]
Using the axe chrome extension, I can see a lot of errors relating to missing landmarks on the site. Many users will rely on landmark regions to navigate to content (e.g. go straight to main content, to navigation, etc).

We should review the landmarks used on each page, with the aide of the axe extension, and make sure all of our content is contained within landmarks, and that a `main` landmark exists on every page.

![screenshot of the axe inspector output with the sections for landmarks highlighted, comprising 11 errors](https://user-images.githubusercontent.com/20773163/124903401-d6d44700-dfdb-11eb-8dfa-c4513a6c5d1c.png)
